src/editors/containers/ProblemEditor/data/OLXParser.js

```diff
@@ -341,10 +341,14 @@ export class OLXParser {
       return ProblemTypeKeys.ADVANCED;
     }
     // make sure compound problems are treated as advanced
-    if (intersectedProblems.length > 1) {
+    // TODO: Find a way to add answers using additional_answers v/s numericalresponse
+    if ((problemTypeKeys.length > 1)
+      || (problemTypeKeys[0] !== ProblemTypeKeys.NUMERIC // multiple numeric problems are really just multiple answers
```

Comment:
1. The example given in the ticket filed isn't "multiple answers?", it is multiple response fields? 
https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/numerical_input.html#add-multiple-correct-responses-in-the-advanced-editor.

Reply:
See `parseNumericResponse()` in this file. Also see `numericInputWithFeedbackAndHintsOLXException` in `olxTestData.js`, which is used in `ReactStateOLXParser.test.js`.

Reply:
Oh Ok interesting, that is sufficient for me, but odd, I guess. OLX stays whacky